src/components/ConversionFormOverlay/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import { Upload, Select, Divider, Button } from "antd";
+
+import styles from "./style.css";
+import theme from "../theme/light-theme.css";
+import classNames from "classnames";
+
+interface ConversionFormOverlayProps {
+    [key: string]: any;
+    active: boolean;
+}
+
+const selectOptions = [
+    { value: "cytosim", label: "cytosim" },
+    { value: "cellPACK", label: "cellPACK" },
+    { value: "Smoldyn", label: "Smoldyn" },
+    { value: "SpringSaLaD", label: "SpringSaLaD" },
+];
+
+const ConversionFormOverlay =
+    ({}: ConversionFormOverlayProps): JSX.Element | null => {
+        const [isFileLoaded, setFileLoaded] = useState<boolean>(false);
+
+        const conversionFormOverlay = (
+            <div className={classNames(styles.container, theme.lightTheme)}>
+                <h3 className={styles.title}>Import a non-native file type</h3>
+                <h3>
+                    Convert and import a non-simularium file by providing the
+                    following information
+                </h3>
+                <h3 className={styles.provide}>
+                    {" "}
+                    Provide file information (required){" "}
+                </h3>
+                <h3 className={styles.selectTitle}>Simulation Engine</h3>
+                <div className={styles.uploadContainer}>
+                    <Select
+                        className={styles.selectorBox}
+                        bordered={true}
+                        defaultValue="Select"
+                        options={selectOptions}
```

Comment:
we are going to set this in redux state in order to request the appropriate template